src/ProgramSerializer.test.js

```diff
@@ -43,5 +55,7 @@ test('Roundtrip program', () => {
         {block: 'left45'}, {block: 'left90'}, {block: 'left180'},
         {block: 'right45'}, {block: 'right90'}, {block: 'right180'}
     ];
```

Comment:
For the roundtrip test, use the same program as used above:

```
{block: 'forward1'}, {block: 'forward2'}, {block: 'forward3'},
{block: 'backward1'}, {block: 'backward2'}, {block: 'backward3'},
{block: 'left45'}, {block: 'left90'}, {block: 'left180'},
{block: 'right45'}, {block: 'right90'}, {block: 'right180'},
{block: 'startLoop', iterations: 2, label: 'A'},
{block: 'endLoop', label: 'A'}
```

To test all of the blocks.